plugins/chunter-resources/src/components/ChannelScrollView.svelte

```diff
@@ -800,7 +800,9 @@
   }
 
   .ref-input {
+    flex-shrink: 0;
     margin: 1.25rem 1rem 1rem;
+    max-height: 18.75rem;
```

Comment:
Maybe increase it a little? `26rem` or something like
with attachments the input is too small for me

Reply:
Changed to 26rem